usr/src/lib/libcmdutils/common/taskq.c

```diff
@@ -28,7 +28,18 @@
  * Copyright (c) 2014 by Delphix. All rights reserved.
  */
 
-#include <sys/zfs_context.h>
+#include <thread.h>
+#include <synch.h>
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/debug.h>
+#include <sys/sysmacros.h>
+
+#include "taskq.h"
+
+/* Maximum percentage allowed for TASKQ_THREADS_CPU_PCT */
+static int taskq_cpupct_max_percent = 1000;
```

Comment:
Where does this number come from? Is this just taking the kernel's number and saying it's likely good enough? Does TASKQ_THREADS_CPU_PCT have much meaning for userland?